src/app.py

```diff
@@ -1,146 +1,36 @@
-import json
-import os
-import random
-import requests
+import dao
+import divvy
+import slack
 import sys
-import time
 
 
-bearer_token = 'Bearer ' + os.getenv('API_KEY')
 GROUP_SIZE = 4
-IMAGE_URL = 'https://raw.githubusercontent.com/cuappdev/bagel-bot/master/bagel-logo.png'
-SLACK_API = 'https://slack.com/api/'
-TESTING_CHANNEL_ID = 'CTF81MFH6'
 
 
-def slack_pr(endpoint, data):
-    return requests.post(
-        SLACK_API + endpoint,
-        data = json.dumps(data),
-        headers = {
-            'Authorization': bearer_token,
-            'Content-type': 'application/json'
-        }
-    ).json()
+def create_bagel_instance():
+    bagelers = slack.bagelers_get()
+    print(bagelers)
+    users = [dao.user_fetch(slack_id, slack.name_get(slack_id)) for slack_id in bagelers]
```

Comment:
if `bagelers` is a list of `slack_id`'s, then I feel like the variable name should be `bageler_slack_ids` to be more representative